packages/@react-spectrum/s2/src/TableView.tsx

```diff
@@ -779,20 +786,29 @@ function ResizableColumnContents(props: ResizableColumnContentProps) {
           </div>
           <Chevron size="M" className={chevronIcon} />
         </Button>
-        <Menu onAction={onMenuSelect} items={items} styles={style({minWidth: 128})}>
-          {(item) => <MenuItem>{item?.label}</MenuItem>}
+        <Menu onAction={onMenuSelect} styles={style({minWidth: 128})}>
+          {items.length > 0 && (
+            <MenuSection aria-label={stringFormatter.format('table.standardColumnMenu')}>
+              <Collection items={items}>
+                {(item) => <MenuItem>{item?.label}</MenuItem>}
+              </Collection>
+            </MenuSection>
+          )}
+          {menuItems}
```

Comment:
What happens if the `menuItems` are not wrapped in a `MenuSection` by the user? Does that still work?

Reply:
It still works, dividers for the sections wouldn't be there, but that's the only thing. Since the default ones we add are in a section though, people don't need to worry about forgetting to be divided from those